lms/templates/staff_problem_info.html

```diff
@@ -51,9 +51,9 @@
     </form>
 
   </div>
-</section>
+</div>
 
-<section aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
+<div aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
```

Comment:
In addition to changing to a div, and role="group" per https://openedx.atlassian.net/browse/TNL-5671.